.eslintrc.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
   env: {
     browser: true
   },
-  rules: {},
+  rules: {
+    "ember/no-new-mixins": "warn"
```

Comment:
Isn't there a new rule "no-jquery" which is in the default set?

Reply:
No, that is in 3.13. But I added it anyways.

Reply:
Why aren't you upgrading to 3.13?

Reply:
3.12 is the LTS

Reply:
Updated to latest